reo/src/reopt_xpress_model.jl

```diff
@@ -1,9 +1,9 @@
 using JuMP
 using Xpress
 
-function reopt_model(MAXTIME)
+function reopt_model(MAXTIME, LOGFILE)
    
-    REopt = direct_model(Xpress.Optimizer(MAXTIME=-MAXTIME, OUTPUTLOG = 0))
+    REopt = direct_model(Xpress.Optimizer(MAXTIME=-MAXTIME, logfile=LOGFILE))
```

Comment:
@TK-23 We do not want the logfile enabled in any production branches (will consume too much memory).